### PR TITLE
Golang aha

### PR DESCRIPTION
**Pull request by plorenz**

### CODE REVIEW

File: docfx_project/articles/golang-aha/article.md [hunk 1]
```diff
@@ -0,0 +1,508 @@
+# Golang Aha! Moments
+
+## Introduction
+As we (the OpenZiti team) has progressed on our Go journey, we've stumbed on various
```
Comment (dovholuknf) [inline docfx_project/articles/golang-aha/article.md]: probably just remove the 'has'?

File: docfx_project/articles/golang-aha/article.md [hunk 2]
```diff
@@ -0,0 +1,508 @@
...
+obstacles, settled on some best practices and hopefully gotten better at writing Go
+code. This document is share some of the stumbling blocks and solutions, both for
```
Comment (dovholuknf) [inline docfx_project/articles/golang-aha/article.md]: "document is meant to share"? it's just missing a certain something/word :)

File: docfx_project/articles/golang-aha/article.md (hunk 1 above)
Comment (plorenz) [reply docfx_project/articles/golang-aha/article.md]: fixed

File: docfx_project/articles/golang-aha/article.md (hunk 2 above)
Comment (plorenz) [reply docfx_project/articles/golang-aha/article.md]: Fixed

**Review by dovholuknf — APPROVED** [review]

**Pull request merged**